src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -2153,9 +2153,15 @@ static String sanitizeFuncotationFieldForVcf(final String individualFuncotationF
         final List<String> badLetters = Arrays.asList(",", ";", "=", "\t", VcfOutputRenderer.HEADER_LISTED_FIELD_DELIMITER, " ", "\n", VcfOutputRenderer.ALL_TRANSCRIPT_DELIMITER);
 
         // Encoded version:
-        final List<String> cleanLetters = badLetters.stream().map(
-                s -> "_%" + String.format("%02X", s.getBytes(StandardCharsets.US_ASCII)[0]) + "_")
-                .collect(Collectors.toList());
+        final List<String> cleanLetters = new ArrayList<>();
+        for(String s: badLetters) {
+            String hex = Integer.toHexString(s.getBytes(StandardCharsets.US_ASCII)[0] & 0xFF).toUpperCase();
```

Comment:
This is great, but I think you can simplify it into an if statement with one assignment in each branch:

```
if (s.getBytes(StandardCharsets.US_ASCII)[0] < 16) {
    // Need to add leading zero
}
else {
    // straight conversion
}
```

Then you can do one assignment (with a couple of `+` operations) in each of those conditions.